doc/articles/uno-build-error-codes.md

```diff
@@ -129,6 +129,18 @@ When building with Rider on Linux or macOS, unsupported target frameworks are [n
 
 See how to [make platforms conditional](xref:Uno.GettingStarted.CreateAnApp.Rider#considerations-for-macos-and-linux) for Rider.
 
+### UNOB0015: The desktop TargetFramework must be placed first
+
+In Visual Studio 17.12 or later, when both mobile (`-ios`, `-android`, `-maccatalyst`) and `desktop` target frameworks are used, the `-desktop` target framework must be placed first in order for WSL debugging to work.
+
+If `-desktop` is not first, the following message will appear:
+
+```text
+The project doesn't know how to run the profile with name 'MyApp (Desktop WSL2)' and command 'WSL2'.
```

Comment:
nit:
```suggestion
The project doesn't know how to run the profile with the name 'MyApp (Desktop WSL2)' and command 'WSL2'.
```

Reply:
This is the actual visual studio message :)